cmd/podman/networks/update.go

```diff
@@ -0,0 +1,58 @@
+package network
+
+import (
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/pkg/domain/entities"
+	"github.com/spf13/cobra"
+)
+
+var (
+	networkUpdateDescription = `Update an existing podman network`
+	networkUpdateCommand     = &cobra.Command{
+		Use:               "update [options] [NAME]",
+		Short:             "update an existing podman network",
+		Long:              networkUpdateDescription,
+		RunE:              networkUpdate,
+		Args:              cobra.MaximumNArgs(1),
+		ValidArgsFunction: completion.AutocompleteNone,
+		Example:           `podman network update podman1`,
+	}
+)
+
+var (
+	networkUpdateOptions entities.NetworkUpdateOptions
+)
+
+func networkUpdateFlags(cmd *cobra.Command) {
+	flags := cmd.Flags()
+
+	addDNSServerFlagName := "add-dns-servers"
```

Comment:
Why add and remove? Why not just allow them to set a new array of nameservers?

Reply:
Underlying API supports update https://github.com/containers/common/pull/1249 so we can add that easily but the customer mentioned that he should be able to `add or remove` DNS servers one reason I could think of is that it prevents from maintaining and fetching a list manually if they want to squeeze in a new DNS server in.

But I agree that top level update option is also helpful and we can add it in this PR or in a different PR just after this whatever maintainers agree with.

Reply:
I think add and remove makes sense for the usability, let's assume I disconnect from a VPN I would want to remove the VPN server without knowing what the others were for example.